CHANGELOG.md

```diff
@@ -26,6 +26,7 @@ This project adheres to [Semantic Versioning](http://semver.org/).
  - #3795, Clarify `Accept: vnd.pgrst.object` error message - @steve-chavez
  - #3779, Always log the schema cache load time - @steve-chavez
  - #3706, Fix insert with `missing=default` uses default value of domain instead of column - @taimoorzaeem
+ - #3697, Handle queries on non-existing table gracefully - @taimoorzaeem
```

Comment:
Seeing the change in the tests output, I think this should also be listed in `### Changed`.

Reply:
> Does that mean you would not consider backpatching this?

Yeah, I was thinking no backpatch. Just to be extra careful.

>  So this is hardly breaking any existing workflows, right?

I know some users that rely on table requests with no resource embedding being fast (in presence of slow relationships loads), if that has changed (test mentioned on https://github.com/PostgREST/postgrest/pull/3869#issuecomment-2605438040) then it would certainly break existing workflows.